This file is part of MXE.
See index.html for further information.

This patch has been taken from:
http://git.tukaani.org/?p=xz.git;a=commit;h=03ed742a3a4931bb5c821357832083b26f577b13

From 03ed742a3a4931bb5c821357832083b26f577b13 Mon Sep 17 00:00:00 2001
From: Lasse Collin <lasse.collin@tukaani.org>
Date: Thu, 19 Apr 2012 14:02:25 +0300
Subject: [PATCH] liblzma: Fix Libs.private in liblzma.pc to include -lrt when
 needed.

---
 src/liblzma/liblzma.pc.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/liblzma/liblzma.pc.in b/src/liblzma/liblzma.pc.in
index 7f11f1a..9fa4891 100644
--- a/src/liblzma/liblzma.pc.in
+++ b/src/liblzma/liblzma.pc.in
@@ -16,4 +16,4 @@ URL: @PACKAGE_URL@
 Version: @PACKAGE_VERSION@
 Cflags: -I${includedir}
 Libs: -L${libdir} -llzma
-Libs.private: @PTHREAD_CFLAGS@ @PTHREAD_LIBS@
+Libs.private: @PTHREAD_CFLAGS@ @LIBS@
-- 
1.8.5.5

